plugins/vpc-shared-eni/plugin/commands.go

```diff
@@ -166,7 +168,15 @@ func (plugin *Plugin) Del(args *cniSkel.CmdArgs) error {
 	err = nb.DeleteEndpoint(&nw, &ep)
 	if err != nil {
 		// DEL is best-effort. Log and ignore the failure.
-		log.Errorf("Failed to delete endpoint, ignoring: %v", err)
+		log.Errorf("Failed to delete endpoint, ignoring: %v.", err)
+	}
+
+	if netConfig.TaskENIConfig.NoInfra {
+		err = nb.DeleteNetwork(&nw)
```

Comment:
Would there be any impact on Fargate with this change, given that Delete Network was not called previously with Fargate but will be called now if no infra is set.

Reply:
I don't think so, but maybe we can all sync up offline for this. Trying to understand where `NoInfra` is being set, and what that exactly entails.

Reply:
As discussed in our syncup, this will not affect my team since we only use the linux implementation of this plugin.

Sid confirmed that these changes are only being made to `bridge_windows`; as long as nothing is changing in `bridge_linux`, I can confidently say should be no conflicts.